docs/Using-Fleet/MDM-setup.md

```diff
@@ -89,9 +89,12 @@ Fleet UI:
 
 2. Follow the on-screen instructions.
 
+### Pending hosts
+Some time after you purchase a Mac through Apple or an authorized reseller, but before it has been set up, the Mac will appear in ABM as in transit. When the Mac appears in ABM, it will also appear in Fleet with **MDM status** set to "Pending." After the new host is set up, the **MDM Status** will change to "On" and the host will be assigned to the default team.
+
 ### Default team
 
-MacOS hosts purchases through Apple or authorized resellers will automatically enroll to the default team in Fleet when they're first unboxed. This means that Fleet will enforce the default team's settings on these hosts.
+All automatically-enrolled hosts will be assigned to a default team of your choosing after they are unboxed and set up. If no default team is set, then the host will be placed in the "No Teams" category. The host will receive the configurations and behaviors set for that team.
 
 > After a host enrolls it can be transferred to a different team. Learn how [here](./Teams.md#transfer-hosts-to-a-team). Transferring a host automatically enforces the new team's settings and removes the old team's settings.
```

Comment:
@zhumo yes it's true but the current sentence might be confusing. You can transfer a host to a not-default team _before or after_ it enrolls.

Because, the "before it enrolls" use case has come up from a customer, I updated the sentence to cover this use case instead.